gammapy/catalog/fermi.py

```diff
@@ -1042,6 +1090,47 @@ def select_source_class(self, source_class):
         return catalog
 
 
+class SourceCatalog4FGL(SourceCatalog):
+    """Fermi-LAT 4FGL source catalog.
+
+    Reference: - https://arxiv.org/abs/1902.10045
```

Comment:
Put it like this with an empty line, so that the formatting in the HTML docs will be OK:
```
References:

- https://arxiv.org/abs/1902.10045
- https://fermi.gsfc.nasa.gov/ssc/data/access/lat/8yr_catalog/
```

Reply:
Done!